ROADMAP.md

```diff
@@ -0,0 +1,12 @@
+---
+title: KubeVirt Roadmap
+---
+
+KubeVirt has implemented [Virtualization Enhancement Proposals (VEPs)](https://github.com/kubevirt/enhancements) in order to agree upon implementation details of large feature work and track the feature lifecycle.
+
+You can view the current [KubeVirt Enhancements Tracking board](https://github.com/orgs/kubevirt/projects/15) to get an idea on the list of currently open VEPs, their state, and whether they have been prioritised by the community for the next release. You are also welcome and encouraged to add to the discussion on these proposals, and to express support for VEPs that you would like to see prioritised.
```

Comment:
Will there be automation to update this link?

Reply:
Ideally yes but I haven't explored it yet. After this is merged I can create an issue on project-infra.